DataFormats/MuonReco/interface/ME0Muon.h

```diff
@@ -0,0 +1,102 @@
+#ifndef MuonReco_ME0Muon_h
+#define MuonReco_ME0Muon_h
+/** \class reco::ME0Muon ME0Muon.h DataFormats/MuonReco/interface/ME0Muon.h
+ *  
+ * \author David Nash NEU
+ *
+ *
+ */
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/Math/interface/AlgebraicROOTObjects.h"
+
+#include <DataFormats/GEMRecHit/interface/ME0SegmentCollection.h>
+
+namespace reco {
+ 
+  class ME0Muon {
```

Comment:
Is there a good reason for this class not to inherit from a RecoCandidate (same as reco::Muon)?